### DISCUSSION TITLE
I cannot decode my file.

### DISCUSSION THREAD

**Discussion by code-wanderlust** [fraunhoferhhi/vvenc#269] [Q&A]
Hello. 

I'm really appreciate to find this repository. It's a great work !

Anyways, I encode my images using this encoder but I don't know how to decode the files. 

After I make project, I only find the 'vvencapp', 'vvencFFapp', 'vvencinterfacetest' , 'vvenclibtest'. 

There is no decoder files. 

So, I try to decode files using original VVC (VTM) decoder  but It didn't work. 

What should I do?

**Comment by adamjw24** [comment]
Give some more info about why it didnt work. Samples, error messages, anything. Should have worked.

Theres also the VVdeC project.

**Reply by code-wanderlust** [reply]
Actually, I didn't know about the vvdec project. Thank you so much.

**Comment by code-wanderlust** [comment]
Here is my command for decode my file : 
./DecoderAppStatic -b /home/tae0.jang/project/image_restoration_proj/codec/vvenc/bin/release-static/out_bin_file_slower.bin -o out.yuv

Also, here is my command line for encoding yuv file. 
./vvencFFapp --inputFile /subdisk/dataset/tyjang/ImgSeqGen/__experiment/screw/vvc_nn_1/screw_inter_0_QP_30/yuv_data/16803224056202114_000000034.yuv --Size 1024x1024 -b out_bin_file_slower.bin --preset slower --QP 30 --FrameRate 1

------------------------
Below is an error message while decode my file in VTM. 

VVCSoftware: VTM Decoder Version 19.2 [Linux][GCC 7.5.0][64 bit] [SIMD=AVX2]
POC    0 LId:  0 TId: 0 ( IDR_N_LP, I-SLICE, QP 29 ) [DT  0.040] [L0] [L1] [:,(unk)]
POC    4 LId:  0 TId: 3 ( STSA, B-SLICE, QP 38 ) [DT  0.031] [L0 0] [L1 0c] [:,(unk)]
POC    2 LId:  0 TId: 4 ( STSA, B-SLICE, QP 37 ) [DT  0.034] [L0 0 4] [L1 4c 0] [:,(unk)]
POC    1 LId:  0 TId: 5 ( STSA, B-SLICE, QP 39 ) [DT  0.028] [L0 0 2] [L1 2c 4] [:,(unk)]
POC    3 LId:  0 TId: 5 ( STSA, B-SLICE, QP 39 ) [DT  0.033] [L0 2 0] [L1 4c 2] [:,(unk)]
POC    6 LId:  0 TId: 4 ( STSA, B-SLICE, QP 37 ) [DT  0.033] [L0 4 0] [L1 4c 0] [:,(unk)]
POC    5 LId:  0 TId: 5 ( STSA, B-SLICE, QP 39 ) [DT  0.030] [L0 4c 0] [L1 6 4] [:,(unk)]
POC    7 LId:  0 TId: 5 ( STSA, B-SLICE, QP 39 ) [DT  0.023] [L0 6 4] [L1 6c 4] [:,(unk)]

 Total Time:        0.364 sec.
corrupted size vs. prev_size
stopped (core dumped)

**Reply by adamjw24** [reply]
So you are encoding pictures or videos? Could you post the log of the encoding? How are you generating the yuv files?

Best, Adam
